test/libsolidity/semanticTests/calldata/calldata_array_dynamic_bytes.sol

```diff
@@ -67,6 +67,7 @@ contract C {
 
 // ====
 // compileViaYul: false
+// bytecodeFormat: legacy
```

Comment:
> ```c++
> // via yul disabled because of stack issues.
> ```

Can you try to remove `compileViaYul: false` here? The code does compile on 0.8.29 for me. If it still fails, I'd also try `onlyOnEOF`.

Reply:
Same for `negative_stack_height.sol`.